docs/querying/api.md

```diff
@@ -336,6 +336,75 @@ $ curl http://localhost:9090/api/v1/label/job/values
 }
 ```
 
+## Querying exemplars
+The following endpoint returns a list of exemplars for a valid PromQL query during a specific time range:
+
+```
+GET /api/v1/query_exemplar
+POST /api/v1/query_exemplar
+```
+
+URL query parameters:
+
+- `query=<string>`: Prometheus expression query string.
+- `start=<rfc3339 | unix_timestamp>`: Start timestamp.
+- `end=<rfc3339 | unix_timestamp>`: End timestamp.
+
+Exemplar storage is not part of Prometheus' TSDB, so this endpoint doesn't allow filtering based on time.
```

Comment:
Then you probably want to remove the start and end params?

Reply:
I should just update this comment rather, I added the start and end a few days later after realizing exemplar queries are most likely going to be executed alongside a `query_range` query associated with a grafana dashboard. If there's additional use cases that come up for other query endpoint options/implementations we can discuss those but I've been building the current endpoints implementation with the assumption that the dashboard panel -> exemplars query is the most likely path.

Reply:
This is still unaddressed then